src/modules/data/modelData_gltf.c

```diff
@@ -530,6 +530,11 @@ bool lovrModelDataInitGltf(ModelData** result, Blob* source, ModelDataIO* io) {
     }
   }
 
+  if (model->blobCount && !model->blobs) {
```

Comment:
This broke all newModel calls for me, since `model->blobs` is always NULL here.  For me, the model is importing fine but there's a crash during rendering.  The vertex buffer was NULL during the vertex animation.  I pushed 49e90aa39811080631878f427d6991b23928165e to just skip vertex animation when there isn't a vertex buffer.

Reply:
Ah sorry. I first tried to assert on `model->primitiveCount` but then the cleanup would crash when releasing blobs, because I did it prematurely. Your method is cleaner.

Reply:
Ah I see.  I think there is still an issue here with the error/cleanup code.  Since `blobCount`/`imageCount` are set before `blobs`/`images` are assigned, if there is an error then `lovrModelDataDestroy` is going to try to free non-existent blobs/images.  I'll see if I can improve it.

Reply:
Fixed up in 949b0bae9e3d0fc9f3631c31ef86b32c0a9614fe, just need to ensure arrays exist before releasing their contents